specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2365,7 +2365,8 @@
         "LateralMovement",
         "Collection",
         "Exfiltration",
-        "CommandAndControl"
+        "CommandAndControl",
```

Comment:
As a preview API this is ok. This wouldn't be a breaking change if modelAsString was true, consider changing this in the GA version.
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo